pkg/common/salt-master.service

```diff
@@ -8,6 +8,7 @@ LimitNOFILE=100000
 Type=notify
 NotifyAccess=all
 ExecStart=/usr/bin/salt-master
+User=salt
```

Comment:
How will this accomodate users which don't use 'salt' as the user, I know of one large customer who uses a different name for the user.

Assuming salt is incorrect and have already had to fix that assumption for log rotation.

Question then is when will the value for 'User' get set, since have to read the configuration file etc. to know what user is set to, and then update the systemd service file, which is part of the install process, so installed before configuration read.  Implies a need to update service file and restart with the new value, and additional checks in case the user changes the configuration value for 'user' too

Reply:
this will cause issues with people using the 'root' user for pam auth funtionality